analyzers/tests/SonarAnalyzer.UnitTest/TestCases/OperatorsShouldBeOverloadedConsistently.CSharp11.cs

```diff
@@ -0,0 +1,28 @@
+﻿public class Foo : IEquality<Foo> // Noncompliant FP Repro for: https://github.com/SonarSource/sonar-dotnet/issues/6442
```

Comment:
This is not an FP. See my comment here https://github.com/SonarSource/sonar-dotnet/issues/6442#issuecomment-1332174804

Please remove this sample.

Reply:
Done and the issue is closed.